terraform/Makefile

```
@@ -25,7 +25,8 @@ common-tests: $(TERRAFORM_PROJECT_DIR)/.terraform/plugins/selections.json ## Exe
 	@cd $(CURDIR)/tests/ && go mod download && go test -v -timeout 30m
 
 plan: lint $(TERRAFORM_PROJECT_DIR)/.terraform/plugins/selections.json ## Write the planned changes into $(PLAN_FILE_NAME) (but do NOT apply them)
-	@cd $(TERRAFORM_PROJECT_DIR) && terraform plan -compact-warnings -lock=false -no-color > $(TERRAFORM_PROJECT_DIR)/$(PLAN_FILE_NAME)
+	@cd $(TERRAFORM_PROJECT_DIR) && terraform plan -compact-warnings -lock=false -out=tfplan &> /dev/null
```

```suggestion
	@cd $(TERRAFORM_PROJECT_DIR) && terraform plan -compact-warnings -lock=false -out=tfplan > /dev/null
```

or eventually `1> /dev/null` but not sure about the `&>` notation (it means redirecting "1" to "1" 🤔 )